feat(serverless): add PN info (#3739)
scaleway/docs-content: faq/serverless-containers.mdx

## How does Serverless Container healthcheck work ?

A Serverless Container is set to `ready` once the specified port is correctly bound to the container, and will start receiving traffic. If your application needs to perform some tasks before receiving traffic (e.g. connect to a database), it's important to run them before binding to the port (starting the webserver).
For now, the `HEALTHCHECK` Docker directive has no impact on container readiness. In the future, the healthcheck will be customizable for your applications.

> ## How to configure access to a Private Network ?

Reviewer: ```suggestion
## How can I configure access to a Private Network?
```

> Scaleway Serverless Containers does not support Virtual Private Networks (VPC) for now, it's a feature under development.

Reviewer: ```suggestion
Scaleway Serverless Containers does not currently support Scaleway VPC or Private Networks, though this feature is under development.
```

> If it's possible to add some network restrictions on your resource, here is the list of prefixes used at Scaleway: [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/). Serverless ressources does not have dedicated and predictible IP address.

Reviewer: ```suggestion
To add network restrictions on your resource, consult the [list of prefixes used at Scaleway](https://www.scaleway.com/en/peering/). Note that Serverless resources do not have dedicated or predictable IP addresses.
```
